data/db/slr-nikon.xml

```diff
@@ -3060,22 +3060,38 @@
     <lens>
         <maker>Nikon</maker>
         <model>Nikon AF Zoom-Nikkor 28-200mm f/3.5-5.6G IF-ED</model>
-        <model lang="en">Nikkor AF 28-200mm f/3.5-5.6G IF-ED</model>
```

Comment:
Please restore this line.  The `<model>` line is used for automatic matching; therefore, I should be what exiv2 says.  In contrast, the `<model lang="en">` line is for the human eye.  For Nikon lenses, Lensfun removed “Nikon” and “Zoom” to make it more compact.